bigchaindb/pipelines/block.py

```diff
@@ -73,10 +74,15 @@ def validate_tx(self, tx):
 
         # If transaction is not valid it should not be included
         try:
+            if tx.operation == Transaction.GENESIS:
+                blocks_count = backend.query.count_blocks(self.bigchain.connection)
```

Comment:
Yes. When the pipeline is started the genesis block already needs to be present.
Things like this https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/pipelines/vote.py#L175 would fail if the genesis block is not there.